2.14/linux/entrypoint/entrypoint.sh

```diff
@@ -2,7 +2,7 @@
 
 if [ -e "${ENTRYPOINT_HOME}/pre_start.sh" ]; then
   echo "Pre-Start Configuration Script found, running now..."
-  chmod 755 ${ENTRYPOINT_HOME}/pre_start.sh
+  sudo chmod 755 ${ENTRYPOINT_HOME}/pre_start.sh
```

Comment:
Look into this:  https://github.com/oconnormi/docker-ddf-base/blob/master/2.14/linux/entrypoint/entrypoint.sh#L12-L13

@oconnormi might not like us to do sudo every where.  We are running our script as a non root user.

Reply:
Ok. Let me change it.